README.md

```diff
@@ -60,20 +60,20 @@ Returns a readable stream of your project's source files. By default, these are
 
 Returns a readable stream of your project's dependencies. This stream is automatically populated based on the files loaded inside of your project. You can include additional dependencies via the `extraDependencies` property in [`ProjectOptions`](src/polymer-project.ts) (this can be useful when the analyzer fails to detect a necessary dependency.)
 
-#### project.analyzer
+#### project.startBuild()
```

Comment:
There's a description of what's happening in the analyzer: https://github.com/Polymer/polymer-build/blob/fbe949bbe39ecc5de40ff8edf036879752bc94fd/src/analyzer.ts#L119

But it doesn't go out of its way to answer this question. I'll reword to be more clear and then reference that in `project.startBuild()`.